.github/release-drafter.yml

```diff
@@ -1,5 +1,48 @@
 # Config for https://github.com/apps/release-drafter
-branches:
-  - master
+name-template: "$RESOLVED_VERSION 🦉 "
+tag-template: "$RESOLVED_VERSION"
+categories:
+  - title: "🚀 New Features and Enhancements "
+    labels:
+      - "feature"
+      - "enhancement"
+  - title: "💣 Breaking Changes "
+    labels:
+      - "breaking-change"
+  - title: "🏇 Optimizations "
+    labels:
+      - "optimize"
+      - "performance"
+  - title: "🐛 Bug Fixes "
+    labels:
+      - "bugfix"
+      - "bug"
+  - title: "🔨 Maintenance "
+    labels:
+      - "maintenance"
+      - "refactoring"
+      - "chore"
+      - "build"
+      - "ci"
+change-template: "- $TITLE (#$NUMBER) @$AUTHOR"
+version-resolver:
+  major:
+    labels:
+      - "major"
+  minor:
+    labels:
+      - "minor"
+  patch:
+    labels:
+      - "patch"
+  default: patch
+exclude-labels:
+  - "skip-changelog"
 template: |
+  > Refer to https://dvc.org/doc/install for installation instructions.
+
+  ## Changes
+
   $CHANGES
+
+  Thanks again to $CONTRIBUTORS for the contributions! 🎉
```

Comment:
weird to `@mention` everyone again a second time. waste of a line?

Reply:
Nope, I don't think it's enough to thank people for their contributions. Maybe add another line too? :stuck_out_tongue_winking_eye:

Reply:
Anyway, they don't get notiifications, so should be fine, I guess.

Reply:
to be clear, each issue/PR changelog entry already has the person `@mention`ed, which is already more than most changelogs do and is also arguably besides the point of a changelog. Having another line at the bottom mentioning everyone again is pointless. At that stage, yes, you may as well  really add another line soliciting additional contributions and linking to https://dvc.org/doc/user-guide/contributing/core